pkg/api/types.go

```diff
@@ -1268,8 +1268,23 @@ func (k *KubernetesConfig) IsIPMasqAgentEnabled() bool {
 	return k.isAddonEnabled(IPMASQAgentAddonName, IPMasqAgentAddonEnabled)
 }
 
-// IsNSeriesSKU returns whether or not the agent pool has Standard_N SKU VMs
-func IsNSeriesSKU(p *Properties) bool {
+// IsNSeriesSKU returns true if the agent pool contains an N-series (NVIDIA GPU) VM
+func (a *AgentPoolProfile) IsNSeriesSKU() bool {
+	return common.IsNvidiaEnabledSKU(a.VMSize)
+}
+
+// HasNSeriesSKU returns whether or not there is an N series SKU agent pool
+func HasNSeriesSKU(p *Properties) bool {
```

Comment:
is this function a duplicate of the below one?

Reply:
it's dead code, will remove